lib/commands/deploynodeapp.js

```diff
@@ -425,6 +517,223 @@ function proxyCreationDone(err, req, body, opts, done) {
   }
 }
 
+function getToken(opts, cb) {
+  var ssoURL = opts.baseuri && opts.baseuri.indexOf('e2e') > -1 ? 'https://login.e2e.apigee.net/oauth/token' : 'https://login.apigee.com/oauth/token'
```

Comment:
Use the same approach that the `get_token` script does by using the `SSO_LOGIN_URL` environment variable.